src/test/suite/fixtures/recorded/lineNumbers/takeRowTwoPastDownThree.yml

```diff
@@ -0,0 +1,35 @@
+spokenForm: take row two past down three
+languageId: typescript
+command:
+  actionName: setSelection
+  partialTargets:
+    - type: primitive
+      selectionType: line
+      modifier:
+        type: lineNumber
+        anchor: {lineNumber: 1, isRelative: false}
+        active: {lineNumber: 3, isRelative: true}
+  extraArgs: []
+marks: {}
+initialState:
+  documentContents: |
+
+    const value = "Hello world";
+
+    const value = "Hello world";
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+finalState:
+  documentContents: |
+
+    const value = "Hello world";
+
+    const value = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 3, character: 28}
```

Comment:
So "down three" is a mark, not a modifier? I guess fine for now but I'd argue that long term we want it as a modifier

Reply:
Yes this is just a stopgap intill we have the full "next" syntax working.